TACC/Core-Portal
# FP-417: Fix link heights (based on FP-389 code)

## Overview: ##

For a [comment](https://github.com/TACC/Frontera-Portal/pull/19#pullrequestreview-410225457) on PR #19, fix height of nav links to:

1. match static UI design (always the goal)
2. use more consistent code (helps for later extraction)

## PR Status: ##

* Work in Progress.
* Hold.

## Related Jira tickets: ##

* [FP-417](https://jira.tacc.utexas.edu/browse/FP-417)

## Summary of Changes: ##

- change vertical height of all nav links to be _approximately_ ¹ `36px` (not `40px`).
- use consistent style rules that affect vertical height of nav links
- use consistent markup (for code relevant to FP-389)
- add comments to explain how non-intuitive units match desired values

¹ Not diving down the rabbit hole that is units, in this PR.

## Testing Steps: ##
1. Load each section.
2. Confirm primary sidebar nav links have height that is near `36px` (`35.19px`).
2. Confirm each section content's nav links (if any) have height that is near `36px` (`35.19px`).
    - ~~Applications secondary nav and app tray, will have `35px`.~~ _As of 2020-05-28, these links also have `35.19px`!_

## Notes: ##

### Testing Since 2020-05-28

The conflict resolution during the merge of `master` needs to be confirmed to have not broken anything (not expected, but gotta make sure).

- [x] Re-test PR #19 (FP-389) (just in Firefox)
- [x] Re-test PR #25 (FP-417)
    - [x] Firefox — Wes, Sal
    - [x] Safari — Sal
    - [x] Chrome — Sal
